packages/resolve-bus/src/index.js

```diff
@@ -1,6 +1,34 @@
-export default function ({ driver }) {
+export default function (createDriver, options) {
```

Comment:
Preserve signature `({ driver })` (as described in https://trello.com/c/I4xbk5GT/96-resolve-bus-initial).